### DISCUSSION TITLE
Future features discussion

### DISCUSSION THREAD

**Discussion by Eldenroot** [Cu8eR/thankyou-like-plugin#306] [General]
I really like this kind of GUI - when oyu click on the list of users you will get detailed info in one place.

![image](https://user-images.githubusercontent.com/778890/100541941-8e83c800-3247-11eb-8cd8-aff849bfe50a.png)

Maybe this could brings us a new discussion about this (and for future of this plugin). I do not mean to add this, just open a discussion, because this could bring us new posibilities (for example most requested feature - like/dislike or in any other word +1/-1). I am not a huge fan of this, but I have promised that we will open a discussion and let check what we can do or not... and you (@lairdshaw ) are the lead coder of this project so your opinion is the most important 👍

**Comment by lairdshaw** [comment]
Hey @Eldenroot. I'm looking to devote my coding time to other projects for the moment, so it's probably best to invite other contributors to discuss/implement features like this. In principle though I have no objection to the addition of a GUI like this, and/or to the addition of the ability to dislike (would there be a similar action for "dis-thanking" though? If so, what would it be called?).

**Comment by Eldenroot** [comment]
@lairdshaw - OK, I see your point :) I hope that you will be still there and lurking around to lend a helping hand or fix some ugly nasty bug :)

About your question - it would be available only for Like/Dislike, thank you will remain the same (so thank you will be the traditional way, like and dislike will be the extended).

**Comment by effone** [comment]
I have enabled the [Discussion](https://github.com/mybbgroup/Thank-you-like-system/discussions) for this repo.
Think that is the right place for this kind ...

Edit: Can't convert this to discussion though. The feature seems to be disabled temporarily.
https://github.com/github/feedback/discussions/2952

**Comment by Eldenroot** [comment]
Ok, good idea!
